src/Autofac/Core/BaseGenericResolveDelegateInvoker.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Autofac Project. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace Autofac.Core;
+
+/// <summary>
+/// Provides a base implementation for the dynamically generated DelegateInvoker classes.
+/// </summary>
```

Comment:
Ugh, get a warning trying to use a cref to the generator; probably because the analyzer project is not a referenced output assembly of the project.  I'll pop it in a remarks block instead without the `<see />` tag.

Reply:
Yeah, that's why I put it in `<c></c>` ("code") tags rather than a `<see />`. I just wanted to illustrate in the docs that it wasn't something arbitrarily capitalized, but is actually a name.

Oh, and I messed up the `<see>` tag, it should be `<seealso>` - you can't have `<see>` outside a main block.